Wabbajack/Views/MainWindow.xaml

```diff
@@ -81,4 +81,11 @@
             </Button>
         </mahapps:WindowCommands>
     </mahapps:MetroWindow.RightWindowCommands>
+    <Window.TaskbarItemInfo>
+        <TaskbarItemInfo x:Name="taskBarItemInfo1" 
+                     ThumbnailClipMargin="80,0,80,140"
+                     Description="Progress"
+                     ProgressValue="{Binding PercentCompleted}">
```

Comment:
So the concepts I mentioned about XAML binding here to MainWindowVM's PercentCompleted seems like it won't work for TaskbarItemInfo, specifically.  I tried it out and it doesn't seem to take.  Seems to be an oddball legacy Winforms control that doesn't support binding?  Either that, or I messed something up.

Either way, it for sure is an oddball control:  It doesn't appear on the control tree of any live XAML inspector or anything.. which makes sense as it's it's own oddball thing:  the taskbar widget/item.

TLDR, i don't think XAML bindings will work.. so this is one of the few times codebehind systems like you were initially using are actually appropriate and required due to it being older tech.

I would suggest a callback something like this inside MainWindow.xaml.cs:
```
 _mwvm.WhenAny(x => x.PercentCompleted)
    .Subscribe(p =>
    {
        this.taskBarItemInfo1.ProgressValue = (double)p;
    });
```

This would likely need to go inside the `Initialized` lambda, as taskbarItemInfo1 isn't initialized until that callback occurs.